--- libcap-ng-0.7.4/bindings/python/Makefile.am.omv~	2014-06-19 14:18:10.459689513 +0200
+++ libcap-ng-0.7.4/bindings/python/Makefile.am	2014-06-19 14:18:19.166689541 +0200
@@ -23,7 +23,7 @@ SUBDIRS = test
 EXTRA_DIST = capng_swig.i
 CONFIG_CLEAN_FILES = *.loT *.rej *.orig
 AM_CFLAGS = -fPIC -DPIC
-PYLIBVER ?= python$(PYTHON_VERSION)
+PYLIBVER ?= python$(PYTHON_VERSION)m
 INCLUDES = -I. -I$(top_builddir) -I/usr/include/$(PYLIBVER)
 LIBS = $(top_builddir)/src/libcap-ng.la
 pyexec_PYTHON = capng.py
--- libcap-ng-0.7.4/configure.ac.omv~	2014-06-19 14:17:41.235689419 +0200
+++ libcap-ng-0.7.4/configure.ac	2014-06-19 14:17:41.240689419 +0200
@@ -123,7 +123,7 @@ if test x$use_python = xno ; then
 else
 AC_MSG_RESULT(testing)
 AM_PATH_PYTHON
-if test -f /usr/include/python${am_cv_python_version}/Python.h ; then
+if test -f /usr/include/python${am_cv_python_version}m/Python.h ; then
 	python_found="yes"
 	AC_MSG_NOTICE(Python bindings will be built)
 else
